docs/examples/custom-json/README.md

```diff
@@ -0,0 +1,307 @@
+    "implementation": "models.JsonHelloWorldModel"
+}
+```
+
+### Start serving our model
+
+Now that we have our config in-place, we can start the server by running `mlserver start .`. This needs to either be ran from the same directory where our config files are or pointing to the folder where they are.
+
+```shell
+mlserver start .
+```
+
+Since this command will start the server and block the terminal, waiting for requests, this will need to be ran in the background on a separate terminal.
+
+### Send test inference request (REST)
+
+
+We now have our model being served by `mlserver`.
+To make sure that everything is working as expected, let's send a request from our test set.
+
+For that, we can use the Python types that `mlserver` provides out of box, or we can build our request manually.
+
+
+```python
+import requests
+import json
+
+inputs = {
+    "name": "Foo Bar",
+    "message": "Hello from Client (REST)!"
+}
+
+# NOTE: this uses characters rather than encoded bytes. It is recommended that you use the `mlserver` types to assist in the correct encoding.
+inputs_string= json.dumps(inputs)
+
+inference_request = {
+    "inputs": [
+        {
+            "name": "echo_request",
+            "shape": [len(inputs_string)],
+            "datatype": "BYTES",
+            "data": [inputs_string]
+        }
+    ]
+}
+
+endpoint = "http://localhost:8080/v2/models/json-hello-world/infer"
+response = requests.post(endpoint, json=inference_request)
+
+response.json()
+```
+
+### Send test inference request (gRPC)
+
+
+Utilizing string data with the gRPC interface can be a bit tricky. To ensure we are correctly handling inputs and
+outputs we will be handled correctly.
+
+For simplicity in this case, we leverage the Python types that `mlserver` provides out of the box. Alternatively,
+the gRPC stubs can be generated regenerated from the V2 specification directly for use by non-Python as well as 
+Python clients.
+
+
+```python
+import requests
+import json
+import grpc
+import mlserver.grpc.converters as converters
+import mlserver.grpc.dataplane_pb2_grpc as dataplane
+import mlserver.types as types
+
+model_name = "json-hello-world"
+inputs = {
+    "name": "Foo Bar",
+    "message": "Hello from Client (gRPC)!"
+}
+inputs_bytes = json.dumps(inputs).encode("UTF-8")
+
+inference_request = types.InferenceRequest(
+    inputs=[
+        types.RequestInput(
+            name="echo_request",
+            shape=[len(inputs_bytes)],
+            datatype="BYTES",
+            data=[inputs_bytes],
+            parameters=types.Parameters(content_type="str")
+        )
+    ]
+)
+
+inference_request_g = converters.ModelInferRequestConverter.from_types(
+    inference_request,
+    model_name=model_name,
+    model_version=None
+)
+
+grpc_channel = grpc.insecure_channel("localhost:8081")
+grpc_stub = dataplane.GRPCInferenceServiceStub(grpc_channel)
+
+response = grpc_stub.ModelInfer(inference_request_g)
+response
+```
+
+## Deployment
```

Comment:
To ensure the example is kept as simple as possible, should we remove the `## Deployment` section altogether? There's now a dedicated page for this, so I'm wondering whether we should consider that outside of the scope of the example (i.e. correct treatment of arbitrary JSON payloads on a custom wrapper).

Reply:
Yes, this is essentially copy and pasted from the other custom model example. I'll remove this section.